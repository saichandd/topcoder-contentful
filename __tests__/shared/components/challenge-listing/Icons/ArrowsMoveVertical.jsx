import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import ArrowsMoveVertical from 'components/challenge-listing/Icons/ArrowsMoveVertical';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();

  renderer.render((
    <ArrowsMoveVertical />
  ));
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
