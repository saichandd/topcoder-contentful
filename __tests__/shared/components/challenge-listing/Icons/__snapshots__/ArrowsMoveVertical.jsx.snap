// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<svg
  height="16px"
  viewBox="0 0 16 16"
  width="16px"
  x="0px"
  xlinkHref="http://www.w3.org/1999/xlink"
  xmlns="http://www.w3.org/2000/svg"
  y="0px"
>
  <g>
    <polygon
      points="13,11 11.6,9.6 9,12.2 9,3.8 11.6,6.4 13,5 8,0 3,5 4.4,6.4 7,3.8 7,12.2 4.4,9.6 3,11 8,16 "
    />
  </g>
</svg>
`;
